### ISSUE TITLE
Deprecated event

### ISSUE THREAD

**Issue by mben42** [zDevelopers/UHPlugin#149]
#### Environment

* **Minecraft version**: 1.11.2
* **Plateform used**: Spigot
* **Bukkit or Spigot version**: git-Spigot-3fb9445-6e3cec8
* **Plugin version**: 1.4
* **Plugin build number**: 85f4301539

#### Problem description

Warning at server start :
[UHPlugin] "UHPlugin v1.4" has registered a listener for org.bukkit.event.player.PlayerAchievementAwardedEvent on method "public void eu.carrade.amaury.UHCReloaded.listeners.GameListener.onPlayerAchievementAwarded(org.bukkit.event.player.PlayerAchievementAwardedEvent)", but the event is Deprecated. "Server performance will be affected"; please notify the authors [Amaury Carrade, azenet, João Roda].

#### Steps to reproduce

Start spigot with the plugin

**Comment by AmauryCarrade** [comment]
Indeed. Thanks you for the report, I'll fix this while migrating the plugin to 1.12.
The performances impact should be minimal.

I can't give you an ETA, though :( Except « before 2018 ».

**Comment by mben42** [comment]
Aypierre-style ETA ;)

I haven't seen any problem with performances at the moment, my CPU and RAM are largely enough.

**Comment by AmauryCarrade** [comment]
_Before 2018_ he said. Anyway it's fixed in 2.0.
